worldeditadditions/utils/selector_helps.lua

```diff
@@ -0,0 +1,9 @@
+function worldeditadditions.player_axis2d(name)
```

Comment:
Generally with utility functions like this I like to add a description so that I know what they do. I follow a convention somewhat similar to Javascript docblocks, and it goes on the line before the function name. In this case, it might look something like this:

```lua
--- Insert a basic description of the command here.
-- An extended description goes here.
-- It may stretch over multiple lines.
-- @param	name	string	The name of the player to operate on.
-- @return	TYPE_OF_RETURN_VALUE	Description of return value
```

Reply:
resolved in commit.